Automattic/jetpack
# My Jetpack: Add Connected Product Card stories

This PR adds stories for the new `<ConnectedProductCard />`:
* Add stories section
* Improve mock data
* Mock product `POST` request (activate/deactivate)

Also, it fixes a uncomfortable issue when checking other products cards

Fixes #

#### Changes proposed in this Pull Request:

* My Jetpack: Add Connected Product Card stories

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Use Storybook
* Confirm you see the story for the ConnectedProductCard
* Confirm you can switch the product by selecting the product slug
* Confirm you can change (once 😅 ) the state of the product
* so on...

https://user-images.githubusercontent.com/77539/156827933-dc0534d9-fc96-421e-bb5d-5e9e48e67aad.mov

* Similarly, you can change the status from the Product Cards Section


https://user-images.githubusercontent.com/77539/156828325-3ad45218-532d-4852-babb-d58a3fdda5c9.mov
